ffi/diplomat/tests/missing_apis.txt

```diff
@@ -1,51 +1,22 @@
-icu::collections::codepointinvliststringlist::CodePointInversionListAndStringList::has_strings#FnInStruct
-icu::collections::codepointinvliststringlist::CodePointInversionListAndStringList::size#FnInStruct
-icu::collections::codepointinvliststringlist::CodePointInversionListAndStringList::try_from#FnInStruct
-icu::collections::codepointinvliststringlist::CodePointInversionListAndStringListError#Enum
-icu::collections::codepointinvliststringlist::Error#Enum
-icu::displaynames::DisplayNamesOptions#Struct
-icu::displaynames::Fallback#Enum
-icu::displaynames::LanguageDisplay#Enum
-icu::displaynames::LanguageDisplayNames#Struct
-icu::displaynames::LanguageDisplayNames::of#FnInStruct
-icu::displaynames::LanguageDisplayNames::try_new_unstable#FnInStruct
-icu::displaynames::RegionDisplayNames#Struct
-icu::displaynames::RegionDisplayNames::of#FnInStruct
-icu::displaynames::RegionDisplayNames::try_new_unstable#FnInStruct
-icu::displaynames::Style#Enum
-icu::displaynames::displaynames::LanguageDisplayNames#Struct
-icu::displaynames::displaynames::LanguageDisplayNames::of#FnInStruct
-icu::displaynames::displaynames::LanguageDisplayNames::try_new_unstable#FnInStruct
-icu::displaynames::displaynames::RegionDisplayNames#Struct
-icu::displaynames::displaynames::RegionDisplayNames::of#FnInStruct
-icu::displaynames::displaynames::RegionDisplayNames::try_new_unstable#FnInStruct
-icu::displaynames::options::DisplayNamesOptions#Struct
-icu::displaynames::options::Fallback#Enum
-icu::displaynames::options::LanguageDisplay#Enum
-icu::displaynames::options::Style#Enum
-icu::displaynames::provider::LanguageDisplayNamesV1#Struct
-icu::displaynames::provider::LanguageDisplayNamesV1Marker#Struct
-icu::displaynames::provider::RegionDisplayNamesV1#Struct
-icu::displaynames::provider::RegionDisplayNamesV1Marker#Struct
+# This file contains all APIs that are exposed in Rust but not over FFI, and not allowlisted in IGNORED_PATHS in tools/ffi_coverage/allowlist.rs.
+# It is generated by `cargo make diplomat-coverage`.
+#
+# If adding an API that expands this file, please consider:
+#
+#  - Whether you can expose that API over FFI in the same PR (and if you do so, be sure to add a `rust_link` annotation)
+#  - Whether the API already has its functionality exposed over FFI (if so, add a potentially-`hidden` `rust_link` annotation to the corresponding FFI API)
+#  - Whether that API is rust-specific functionality that need not be exposed (if so, add it to the allowlist with a note, or add a `hidden` `rust_link` annotation to a related API if possible)
+#  - Whether that API should be punted for later in FFI (if so, please check in with @Manishearth, @robertbastian, or @sffc)
+#  - Whether the API is experimental (if so, add it to the allowlist as experimental)
+#
+# It is acceptable to temporarily have APIs in this file that you plan to add in a soon-upcoming PR.
+#
+# Please check in with @Manishearth, @robertbastian, or @sffc if you have questions
```

Comment:
Nit (optional): Please use a group like `@unicode-org/icu4x-owners` rather than individual names

Reply:
I'd rather do specific names for now since the owners group is not the same group as the people doing FFI.